extensions/src/AWSSDK.Extensions.NETCore.Setup/Properties/AssemblyInfo.cs

```diff
@@ -16,5 +16,9 @@
 [assembly: ComVisible(false)]
 
 
+#if NETFRAMEWORK
```

Comment:
AWSSDK.Extenstions.NETCore.Setup doesn't have any .NET Framework targets so you can get rid of the `#if` and just always set the `AssemblyVersion` to the same as `AssemblyFileVersion`.